tokio/src/util/linked_list.rs

```diff
@@ -164,46 +164,6 @@ impl<T: Link> LinkedList<T> {
     }
 }
 
-cfg_sync! {
-    impl<T: Link> LinkedList<T> {
-        /// Splits this list off at `node`, returning a new list with `node` at its
-        /// front.
-        ///
-        /// If `node` is at the the front of this list, then this list will be empty after
-        /// splitting. If `node` is the last node in this list, then the returned
-        /// list will contain only `node`.
-        ///
-        /// # Safety
-        ///
-        /// The caller **must** ensure that `node` is currently contained by
-        /// `self` or not contained by any other list.
-        pub(crate) unsafe fn split_back(&mut self, node: NonNull<T::Target>) -> Self {
```

Comment:
I think this can still be useful. You just still need to follow ownership rules.

If you however don't want dead code it can certainly be removed.

Reply:
I removed it just to avoid the dead code. We could add an `allow` attribute instead, but I thought that if we needed it again, it would be in the git history. @carllerche any thoughts?

Reply:
Remove is fine for now. We can always find it in the history or grab it from somewhere else. I don't care much either way. It could be flagged as dead code and removed eventually.